aiohttp_datadog/__init__.py

```diff
@@ -24,5 +25,8 @@ def __init__(self, app_prefix, datadog_kwargs=None):
             'http_path:{0}'.format(request.path),
             'request_type:{0}'.format(request.GET.getone('type', None)),
         ]
+        if self.tags:
+            tags.extend(self.tags)
```

Comment:
Adding this logic raises a few questions. What if this creates duplicate tags? What if some of `self.tags` are invalid? And there's probably more considerations we didn't consider yet.

But good news, the datadog library already has this feature properly implemented! We can just let them take care of it. You can set [a `constant_tags` parameter on the `Dogstatsd` client](https://github.com/DataDog/datadogpy/blob/ce6fdc450fe7f2f168da0b1483d52a76b0a757ea/datadog/dogstatsd/base.py#L51).

But if we're gonna let people set this option, it probably makes sense to not handle `constant_tags` specially, and instead just let people pass in any option the same way we have `datadog_kwargs` now. But to use those args we need to create a Dogstatsd client instead of using the global `initialize` function… which is way nicer anyway! So, please do it that way :D

Or, I could write that implementation, if you want.